cmd/check.go

```diff
@@ -105,3 +106,30 @@ var apacheReqCmd = &cobra.Command{
 		return nil
 	},
 }
+
+var blackfireReqCmd = &cobra.Command{
+	Use:   "blackfire",
+	Short: "Test if environment is correctly set for Blackfire",
+	Long: `Will check if your environment variables contains BLACKFIRE_SERVER_ID`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if os.Getenv("BLACKFIRE_SERVER_IDA") == "" {
+			return errors.New(`		/!\\ ERROR /!\\
+		It seems that your environment is not set properly
+		First, make sure you are not using this command with 'sudo'.
+		To fix this issue, run the following commands:
+
+		dm env add --variable=BLACKFIRE_SERVER_ID --value=<YOUR_SERVER_ID>
+		dm env add --variable=BLACKFIRE_SERVER_TOKEN --value=<YOUR_SERVER_TOKEN>
+		dm env add --variable=BLACKFIRE_CLIENT_ID --value=<YOUR_CLIENT_ID>
```

Comment:
do you actually want the client tokens in your local env? don't you just need them on the machine/container that has the probe installed?